### PR TITLE
modify form structure (into one form) & implement form checking function 

### PR DESCRIPTION
**Pull request by barry800414**

### CODE REVIEW

**Comment by qcl** [comment]
This pull request cannot be merged, please resolve the conflicts first, thanks.

**Comment by qcl** [comment]
:shipit:

File: js/grayscale.js [hunk 1]
```diff
@@ -99,7 +100,13 @@ window.fbAsyncInit = function() {
 
     $("#submit").click(function(e) {
         e.preventDefault();
-        checkForm();
+        var msg = checkForm();
+        if(msg == 'success'){
+            submitForm();
+        }
+        else{
+            alert(msg);
```
Comment (mark86092) [inline js/grayscale.js:108]: Do not use alert, since `alert` may not work on phone

Comment (mark86092) [inline js/grayscale.js:108]: 我想到的方法是，把相對應的欄位變成 danger 樣

**Pull request merged**